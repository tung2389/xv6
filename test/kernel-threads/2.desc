Test concurrency on updating variable