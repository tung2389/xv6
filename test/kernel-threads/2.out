XV6_TEST_OUTPUT: shared_global's value: 12
